config/settings.py

```diff
@@ -20,7 +24,7 @@
 # See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-&2#s+8(@(--w0!4_ynpckazn*huhnern=ht+zu5_my)zf8bgix'
+SECRET_KEY = apilayer_key
```

Comment:
Это не ключ от apilayer, а от django и правильнее их указывать сразу (без доп переменных)
```suggestion
SECRET_KEY = os.getenv('SECRET_KEY')
```